documentation/jetty-documentation/src/main/asciidoc/programming-guide/maven/jetty-maven-plugin.adoc

```diff
@@ -11,7 +11,7 @@
 // ========================================================================
 //
 
-[[jetty-maven-plugin]]
+[[jetty-ee10-maven-plugin]]
```

Comment:
this could be ee9 or ee8 as well. 
Maybe something more generic?

Reply:
But jetty does 12 not accept a generic `jetty-maven-plugin`.